### PR TITLE
changed preflight check of logfiles to also include multiple logfiles…

### PR DESCRIPTION
**Pull request by saphjra**
…, not only no logfiles as described in #144

### CODE REVIEW

**Review by cbueth — APPROVED** [review]

**Review by cbueth — CHANGES_REQUESTED** [review]
Failing preflight tests need to be changed.

File: preprocessing/checks/preflight.py [hunk 1]
```diff
@@ -302,8 +302,8 @@ def _check_sessions(data_collection, groups: dict[str, list[str]]) -> None:
             groups.setdefault("EXPERIMENT_*.txt", []).append(sid)
 
         # 4. DATA_LOGFILE_*.txt
-        experiment_logs = _ci_glob(logfiles, "DATA_LOGFILE_*.txt")
-        if len(experiment_logs) != 1:
+        data_logs = _ci_glob(logfiles, "DATA_LOGFILE_*.txt")
```
Comment (cbueth) [inline preprocessing/checks/preflight.py:309]: This variable was shadowing one from point 3.

**Review by cbueth — COMMENTED** [review]
The tests did not account for the new `DATA_LOGFILE_*.txt` check added in the preflight logic, nor for the `!= 1` semantics that replace the old `== 0` checks. I added the missing logfile to all test fixtures, added a `data_logfile_missing` test scenario, and fixed an unrelated naming inconsistency in `test_preflight_stimulus_dir_empty_with_archive`. See 5b67563.

**Comment by cbueth** [comment]
The previous `!= 1` check included missing and duplicate logfiles into one generic error group. I changed that each logfile type (EXPERIMENT, DATA_LOGFILE, GENERAL_LOGFILE) distinguishes between "missing" and "too many files" with a dedicated `"Multiple *.txt logfiles"` error group that includes the session ID and file count. This also added the `DATA_LOGFILE_*.txt` check that was missing from the original commit, and updated test fixtures and scenarios to cover both missing and duplicate cases.
